#!/usr/bin/env python
PACKAGE = "srv_cli"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("range_min",    int_t,    0, "minimun of range", 0,  0, 50)
gen.add("range_max",    int_t,    0, "maximun of range", 100,  51, 100)
gen.add("digit", int_t, 0, "number of fraction digits of random number", 3, 0, 6)

exit(gen.generate(PACKAGE, "srv_cli", "dynamic_param"))
